Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
```

Comment:
I would write this and the following two lines as:

    RUN cd /opt/ldnode/certs \
     && openssl genrsa 2048 > ssl-key.pem \
     && openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem

This cuts down on extraneous layers in the created image (generally you want each layer to be a "meaningful unit of work" -- this means every group of lines starting with `RUN`).